string name
uint16 pos_x
float64 pos_y
